### ISSUE TITLE
ES-based /cases/ endpoint omits empty lists

### ISSUE THREAD

**Issue by jcushman** [harvard-lil/capstone#1210]
The new ES-based /cases/ endpoint leaves out keys if the values are empty lists. For example, the old endpoint https://api.case.law/v1/db_cases/1000000/?full_case=true includes

```
    "casebody": {
        "data": {
            "attorneys": [],
            "judges": [],
        }
    }
```

... but https://api.case.law/v1/cases/1000000/?full_case=true has no "attorneys" or "judges" keys, though it does have them for other cases if they are populated. We should include the keys even if the values are empty.

(I know that this is happening for "attorneys" and "judges" -- not sure if it applies anywhere else.)

**Comment by ChefAndy** [comment]
Ok, thanks. I'm on it.

**Comment by ChefAndy** [comment]
This may be some other sort of issue... elasticsearch ingest?

https://api.case.law/v1/cases/?cite=1%20Ill.%2017&full_case=true

Here's a case with empty lists for both judges and attorneys.

**Comment by ChefAndy** [comment]
```https://api.case.law/v1/cases/?cite=162%20A.D.2d%20136&full_case=true``` is using the list view while ```https://api.case.law/v1/cases/1000000/?full_case=true``` is using the detail view. The case with ID 1000000 when referred to by the cite will return empty attorneys, etc. In the detail view, it will not. The get_casebody function is getting different cases— one has ```xml```, ```text```, and ```html``` as options with empty lists, and one has only ```text``` without empty lists. I think this has to do with the renderers. I'll check it out.

**Comment by ChefAndy** [comment]
https://github.com/elastic/elasticsearch-dsl-py/issues/460

**Comment by ChefAndy** [comment]
https://github.com/elastic/elasticsearch-dsl-py/issues/1094

**Comment by ChefAndy** [comment]
Looks like the only way to do this for now is to manually add in all empty lists.

**Comment by ChefAndy** [comment]
Addressed in #1243
